apps/api/src/app/webhooks/usecases/get-webhook-portal-token/get-webhook-portal-token.usecase.ts

```diff
@@ -1,17 +1,17 @@
 import { BadRequestException, Inject, Injectable, NotFoundException, Scope } from '@nestjs/common';
 import { EnvironmentRepository } from '@novu/dal';
-import { LogDecorator, SvixClient } from '@novu/application-generic';
+import { LogDecorator } from '@novu/application-generic';
+import { Svix } from 'svix';
+import { generateAppId } from '../../../../../../../libs/application-generic/src/webhooks/utils/app-id';
```

Comment:
Please update the import to use app generic, do it n all the relative imports you've done n this PR

```
import { LogDecorator, generateAppId } from '@novu/application-generic';
```